media/css/mozorg/leadership.less

```diff
@@ -0,0 +1,932 @@
+@import '../pebbles/components/footer.less';
+@import '../pebbles/components/modal.less';
+@import '../pebbles/includes/animations.less';
+
+#masthead {
+    background: @color-mozred;
+    color: #fff;
+
+    .toggle {
+        background-image: url('/media/img/pebbles/icon-menu-light.svg');
+    }
+
+    @media @mq-tablet {
+        a {
+            color: #fff;
+        }
+    }
+}
+
+// Header
+.snapshot-header {
+    display: none;
+    background: #e9dbac url('/media/img/sandstone/grain.png');
+    width: 100%;
+    overflow: hidden;
+    position: relative;
+
+    @media @mq-tablet {
+        display: block;
+        height: 300px;
+    }
+
+    @media @mq-desktop {
+        height: 500px;
+    }
+}
+
+.snap {
+    display: inline-block;
+    background: #fff;
+    padding: 20px 20px 50px 20px;
+    box-shadow: 0 2px 2px 2px rgba(0, 0, 0, .2);
+    position: absolute;
+
+    span {
+        .background-size(230px, 230px);
+        background-color: #e8ebd3;
+        background-position: center center;
+        background-repeat: no-repeat;
+        display: block;
+        height: 230px;
+        outline: 1px solid rgba(0, 0, 0, .2);
+        width: 230px;
+    }
+
+    img {
+        display: block;
+        height: 230px;
+        margin: 0;
+        width: 230px;
+    }
+}
+
+.foreground {
+    display: none;
+
+    @media @mq-tablet {
+        display: block;
+        height: 300px;
+        left: 50%;
+        margin-left: -(@width-tablet / 2);
+        position: absolute;
+        top: 0;
+        width: @width-tablet;
+        z-index: 10;
+
+        .snap {
+            position: absolute;
+        }
+
+        .shot-one { // MSurman
+            left: -12px;
+            top: 6px;
+            transform: rotate(-3deg);
+        }
+
+        .shot-two { // Mitchell
+            left: 225px;
+            top: -3px;
+            transform: rotate(5deg);
+        }
+
+        .shot-three { // CBeard
+            left: 485px;
+            top: 9px;
+            transform: rotate(-6deg);
+        }
+    }
+
+    @media @mq-desktop {
+        height: 500px;
+        margin-left: -(@width-desktop / 2);
+        width: @width-desktop;
+
+        .shot-one {
+            left: 120px;
+            top: 96px;
+        }
+
+        .shot-two {
+            left: 375px;
+            top: 103px;
+        }
+
+        .shot-three {
+            left: 635px;
+            top: 90px;
+        }
+    }
+
+    @media @mq-desktop-wide {
+        margin-left: -(@width-desktop-wide / 2);
+        width: @width-desktop-wide;
+
+        .shot-one {
+            left: 320px;
+            top: 96px;
+        }
+
+        .shot-two {
+            left: 575px;
+            top: 103px;
+        }
+
+        .shot-three {
+            left: 835px;
+            top: 90px;
+        }
+    }
+}
+
+@supports ((animation-name: develop) or (-webkit-animation-name: develop)) and ((filter: invert(40%)) or (-webkit-filter: invert(40%))) {
```

Comment:
Yeap the animation does not seem to work on demo currently, so I suspect this might be the culprit: https://bedrock-demo-craigcook.us-west.moz.works/en-US/about/leadership/